tests/planning/d.spec.lua

```diff
@@ -11,6 +11,8 @@ return function()
 
 	describe("test4", function()
 		it("test5", function()
+			-- Duplicate describe blocks are not merged, so this is not a
+			-- duplicate it block.
```

Comment:
This behavior is surprising to me. I think I need to catch up on the interactions with how we merge describe blocks, since it feels like the describe blocks should've been merged and caused this to be an error.

I don't think frameworks like Mocha or Jest have _any_ implicit describe blocks wrapping tests. That was something we did in TestEZ to make tracking the hierarchy of tests a little easier. That obviously introduces interesting edges.

Reply:
Yeah. Since we merge the describe blocks for directories, you'd expect these to be merged too. And from a test reporting point of view that also makes more sense. This is something I want to fix, but for this PR it's only making sure the documentation matches current behavior.